src/patterns/ec2-app.test.ts

```diff
@@ -123,42 +107,81 @@ describe("the GuEC2App pattern", function () {
     expect(stack).toHaveResource("AWS::CloudWatch::Alarm"); //The shape of the alarm is tested at construct level
   });
 
+  it("requires IP addresses to whitelist if set as a restricted access application", function () {
```

Comment:
Minor: I think `creates the appropriate ingress rules for a restricted access application` might be a more helpful test name.

Reply:
Nice, yeah that reads better.